build/common.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <!-- Packages can have independent versions and only increment when released -->
-    <Version>3.0.36$(VersionSuffix)</Version>
-    <HostStorageVersion>5.0.0-beta.2$(VersionSuffix)</HostStorageVersion>
+    <Version>3.0.37$(VersionSuffix)</Version>
+    <HostStorageVersion>5.0.0-beta.3$(VersionSuffix)</HostStorageVersion>
```

Comment:
This package is about to be published as a GA (non-beta) release, OK to add the changes here?

Reply:
Updated to <HostStorageVersion>5.0.0</HostStorageVersion>